docs/source/reference/modules.rst

```diff
@@ -75,6 +75,7 @@ other cases, the action written in the tensordict is simply the network output.
     AdditiveGaussianWrapper
     EGreedyModule
     EGreedyWrapper
+    OrnsteinUhlenbeckProcessModule
     OrnsteinUhlenbeckProcessWrapper
```

Comment:
```suggestion
    OrnsteinUhlenbeckProcessModule
```
since we want to deprecate it

Reply:
hmm upon reflection, let's keep this one, people may still see it in their code

Reply:
Yeah I'd think it's typically a good idea to keep the old documentation and only remove it at the point when the deprecated feature is actually removed